### DISCUSSION TITLE
docs/docker

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#449] [Q&A]
# docs/docker

http://43.138.148.137:9667/docs/docker.html

**Comment by johnny14889** [comment]
导出：docker save -o <保存路径>/myimage.tar myimage:latest
例如：docker save -o /pathtosave/myimage.tar myimage:tag

导入：docker load -i <路径>/myimage.tar
例如：docker load -i /pathtoload/myimage.tar

**Reply by jaywcjlove** [reply]
@johnny14889  已更新

**Comment by yudongjun** [comment]
支持作者
